kubernetes/resource_kubernetes_deployment_test.go

```diff
@@ -222,6 +224,60 @@ func TestAccKubernetesDeployment_with_security_context_run_as_group(t *testing.T
 	})
 }
 
+func TestAccKubernetesDeployment_with_tolerations(t *testing.T) {
+	var conf api.Deployment
+
+	rcName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	imageName := "redis:5.0.2"
+	operator := "Equal"
+	value := "value"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckKubernetesDeploymentDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesDeploymentConfigWithTolerations(rcName, imageName, &tolerationSeconds, operator, nil),
```

Comment:
FYI tests are complaining about this: https://travis-ci.org/terraform-providers/terraform-provider-kubernetes/builds/534938028#L487 and other things, maybe rebasing went south -- i suspect this is off https://github.com/terraform-providers/terraform-provider-kubernetes/pull/448/files#diff-e0142721a483e44565464d665ebff8ddR177

Reply:
Thanks for the head start. Fixed!